src/utils/aquarius.ts

```diff
@@ -64,7 +64,7 @@ export function transformQueryResult(
   )
   result.totalResults = queryResult.hits.total
   result.totalPages = Math.floor(result.totalResults / size)
```

Comment:
think i found another problem here. We use math.floor to get the page number and if for eg we have total result let's say 41 and size of the search 21 we get the total number of pages 1 and you can't switch to the second page and see the next 20 results. I think we use something like this
```suggestion
  result.totalPages = (result.totalResults / size < 1) ? Math.floor(result.totalResults / size) : Math.ceil(result.totalResults / size)
```